code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeTestConstants.java

```diff
@@ -16,7 +16,7 @@ public class EdgeBridgeTestConstants {
 	private EdgeBridgeTestConstants() {}
```

Comment:
Can you remove the EventSource and EventType strings which are no longer needed as they are taken from Core?